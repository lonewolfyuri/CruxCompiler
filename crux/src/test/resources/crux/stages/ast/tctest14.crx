// invalid variable type

func main() : void {
  var a:void;
  ::println();
}
