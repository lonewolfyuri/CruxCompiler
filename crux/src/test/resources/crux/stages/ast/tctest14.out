ast.DeclarationList(3,0)
  ast.FunctionDefinition(3,0)[Symbol(main), []]
    ast.StatementList(4,2)
      ast.VariableDeclaration(4,2)[Symbol(a)]
      ast.Call(5,2)[Symbol(println)]
